src/components/ChangeData/ChangeProfile.module.scss

```diff
@@ -0,0 +1,58 @@
+@import 'src/styles/color.scss';
+@import 'src/styles/breakpoints.scss';
+
+.formBox {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 2rem;
+
+  @media (min-width: $screen-md-min) {
+    justify-content: center;
+    width: 40%;
+  }
+}
+
+.title {
+  margin-bottom: 1rem;
+  color: $severely-burnt-toast;
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  padding: 2rem 10%;
+  background-color: $hornblende-green;
+  color: $translucent-silk;
+  border-radius: 10px;
+}
+
+.textField {
+  width: 100%;
+  padding: 0.5rem 0;
+  margin-bottom: 1rem;
+  background-color: unset;
+  border: none;
```

Comment:
Why do you need resetting? I expect this to be handled in the text field via some "type." Preferably, you will have only one type of input in the product.